graphic_templates/waffle_chart/css/graphic.less

```diff
@@ -0,0 +1,23 @@
+@import "base";
+@import "labels";
+
+tspan.value {
+	font-weight: bold;
+}
+
+.labels {
+	font-size: 12px;
+}
+
+.tooltip {
+	background-color: white;
```

Comment:
The original file seemed to be in tabs and my additions were the ones with spaces. I've converted all to 4 spaces.